smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/StructureGenerator.kt

```diff
@@ -93,7 +93,9 @@ class StructureGenerator(
             write("@JvmStatic")
             write("fun fluentBuilder(): FluentBuilder = BuilderImpl()")
             write("")
-            write("fun builder(): DslBuilder = BuilderImpl()")
+            // the manual construction of a DslBuilder is mostly to support serde, end users should go through
+            // invoke syntax for kotlin or fluentBuilder for Java consumers
+            write("internal fun builder(): DslBuilder = BuilderImpl()")
```

Comment:
**Question**: Do we need to decorate this with `@InternalApi`?

Reply:
it's generated with `internal` visibility so I don't think so.

Reply:
Oh right. Wasn't thinking.